task-maker-format/src/ioi/task_info.rs

```diff
@@ -117,7 +122,7 @@ impl IOITaskInfo {
                 .join("att")
                 .read_dir()
                 .map(|dir| {
-                    dir.filter(|entry| entry.as_ref().unwrap().file_type().unwrap().is_file())
+                    dir.filter(|entry| is_file_or_symlink(entry.as_ref().unwrap().file_type().unwrap()))
```

Comment:
I assume being a symlink is not enough - rather, either a file or a symlink to a file, right?

Reply:
Fixed, turns out `PathBuf::is_file` already traverses symlinks.